### ISSUE TITLE
Feature request: Add apk install package

### ISSUE THREAD

**Issue by matthdsm** [irods/irods#3440]
Hi,

Would it be possible to add an apk install binary to the package repository? This would enable us to easily create a lightweight docker image for iRODS.

Thanks!
M

**Comment by trel** [comment]
@matthdsm you meant to close this issue?

**Comment by matthdsm** [comment]
yeah, I was cleaning up some stale issues yesterday. We've moved on from iRODS since it was a little to complex and maintenance heavy for our needs.

Thanks for checking.
Cheers
M

**Comment by trel** [comment]
Okay - but I still think this is worth investigating.  I'm going to reopen as Docker deployments are becoming more and more prevalent.
